packages/grid/vpn/tailscale.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh
 
+# iptables --list
 # block all traffic but port 80 on the tailscale0 interface
 # iptables -A INPUT -i tailscale0 -p tcp --dport 80 -j ACCEPT
 # iptables -A INPUT -i tailscale0 -p tcp -j REJECT
+# iptables -A OUTPUT -p tcp --destination-port 4000 -j DROP
+
```

Comment:
We can't just block all traffic except port 80 on the tailscale0 interface because it seems like tailscale might be opening random ports so we should double check this still works with a deny all rule.